src/uct/ib/ud/accel/ud_mlx5.c

```diff
@@ -709,10 +709,28 @@ static ucs_status_t uct_ud_mlx5_iface_create_qp(uct_ib_iface_t *ib_iface,
         return status;
     }
 
+    status = uct_ib_mlx5_txwq_init(iface->super.super.super.worker,
+                                   iface->tx.mmio_mode, &iface->tx.wq,
+                                   qp->verbs.qp);
+    if (status != UCS_OK) {
```

Comment:
no need to undo `uct_ib_mlx5_iface_create_qp`?

Reply:
good catch, fixed